src/main/util/consts.py

```diff
@@ -0,0 +1,168 @@
+    USERNAME = 'username'
+    EVENT_TYPE = 'eventType'
+    EVENT_DATA = 'eventData'
+    PROJECT_NAME = 'project_name'
+    FOCUSED_COMPONENT = 'focusedComponent'
+    CURRENT_FILE = 'currentFile'
+    PSI_PATH = 'PSIPath'
+    EDITOR_LINE = 'editorLine'
+    EDITOR_COLUMN = 'editorColumn'
+    TASK = 'task',
+    ATI_ID = 'atiId'
+
+    @classmethod
+    def activity_tracker_columns(cls):
+        return [cls.TIMESTAMP_ATI.value, cls.USERNAME.value, cls.EVENT_TYPE.value, cls.EVENT_DATA.value,
+                cls.PROJECT_NAME.value, cls.FOCUSED_COMPONENT.value, cls.CURRENT_FILE.value, cls.PSI_PATH.value,
+                cls.EDITOR_LINE.value, cls.EDITOR_COLUMN.value, cls.TASK.value]
+
+
+class ACTIVITY_TRACKER_EVENTS(Enum):
+    ACTION = 'Action'
+    COMPILATION_FINISHED = 'CompilationFinished'
+
+    @classmethod
+    def action_events(cls):
+        return ['Run', 'Rerun', 'RunClass', 'DebugClass', 'ToggleLineBreakpoint', 'Debugger.AddToWatch', 'Debug',
+                'Stop', 'Resume', 'StepInto', 'CompileDirty', 'EditorCopy', 'EditorPaste', 'EditorCut', 'ReformatCode',
+                '$Undo', '$Paste', '$Copy', 'ChooseRunConfiguration', 'CopyElement', 'PasteElement', 'CutElement']
+
+
+class DEFAULT_VALUES(Enum):
+    AGE = 0
+    EXPERIENCE = np.nan
+
+
+class TASK(Enum):
+    PIES = 'pies'
+    MAX_3 = 'max_3'
+    ZERO = 'zero'
+    ELECTION = 'election'
+    BRACKETS = 'brackets'
+    MAX_DIGIT = 'max_digit'
+
+
+class TASKS_TESTS(Enum):
+    TASKS_TESTS_PATH = ROOT_DIR + '/../../resources/tasks_tests/'
+    SOURCE_OBJECT_NAME = 'source'
+    INPUT_FILE_NAME = 'in'
+
+
+class LANGUAGE(Enum):
+    JAVA = 'java'
+    PYTHON = 'python'
+    KOTLIN = 'kotlin'
+    CPP = 'cpp'
+    NOT_DEFINED = 'not_defined'
+
+
+class TEST_RESULT(Enum):
+    INCORRECT_CODE = -1
+    LANGUAGE_NOT_DEFINED = -2
+    # It can be incorrect although, but our checking tools cannot find any errors
+    CORRECT_CODE = 0
+
+
+class FILE_SYSTEM_ITEM(Enum):
+    PATH = 0
+    SUBDIR = 1
+    FILE = 2
+
+
+EXTENSION_TO_LANGUAGE_DICT = {
+    'py': LANGUAGE.PYTHON.value,
+    'java': LANGUAGE.JAVA.value,
+    'kt': LANGUAGE.KOTLIN.value,
+    'cpp': LANGUAGE.CPP.value,
+    '': LANGUAGE.NOT_DEFINED.value
+}
+
+LANGUAGE_TO_MIN_SYMBOLS = {
+    # a = int(input())
+    # print(a)
+    LANGUAGE.PYTHON.value: 25,
+
+    # class A{
+    # public static void main(String args[]){
+    # Scanner in = new Scanner(System.in );
+    # Int a = in.nextInt();
+    # System.out.print(a);}}
+    LANGUAGE.JAVA.value: 140,
+
+    # fun main(args: Array<String>){
+    # val a:Int = readLine()!!.toInt()
+    # print(a)}
+    LANGUAGE.KOTLIN.value: 80,
+
+    # #include <iostream>
+    # int main(){
+    # int a;
+    # cin>>a;
+    # cout<<a;}
+    LANGUAGE.CPP.value: 60
+}
+
+LANGUAGE_TO_OUTPUT = {
+    LANGUAGE.PYTHON.value: "print",
+    LANGUAGE.JAVA.value: "System.out.print",
+    LANGUAGE.KOTLIN.value: "print",
+    LANGUAGE.CPP.value: "cout",
```

Comment:
Sometimes people can use the printf function for output in the console for C++ language. You can see the documentation: http://www.cplusplus.com/reference/cstdio/printf/